client/src/components/dashboard-component/nav-dashboard-component/index.jsx

```diff
@@ -11,41 +13,78 @@ function DashboardNavigationComponent() {
     <NavigationContainer
       $sx={{
         display: "flex",
-        flexDirection: "column",
+        flexDirection: "row",
+        justifyContent: "center",
+        alignItems: "center",
         height: "100%",
       }}
     >
-      <Grid container spacing={2} justifyContent="center">
-        <Grid item xs={12} sm={4}>
+      <Grid container spacing={2} justifyContent="center" direction="row">
+        <Grid item xs={14} sm={2.25}>
           <MainNavButton
             variant="contained"
             startIcon={<PeopleIcon />}
             iconbackgroundcolour="#3568E5"
             onClick={() => navigate("/clients/")}
+            sx={{
+              whiteSpace: "nowrap",
+            }}
           >
             Clients
           </MainNavButton>
         </Grid>
-        <Grid item xs={12} sm={4}>
+        <Grid item xs={14} sm={2.25}>
           <MainNavButton
             variant="contained"
             startIcon={<StarIcon />}
             iconbackgroundcolour="#1AA66A"
             onClick={() => navigate("/job-leads/")}
+            sx={{
+              whiteSpace: "nowrap",
+            }}
           >
             Job Leads
           </MainNavButton>
         </Grid>
-        <Grid item xs={12} sm={4}>
+        <Grid item xs={14} sm={2.25}>
           <MainNavButton
             variant="contained"
             startIcon={<WorkIcon />}
             iconbackgroundcolour="#E53568"
             onClick={() => navigate("/employers/")}
+            sx={{
+              whiteSpace: "nowrap",
+            }}
           >
             Employers
           </MainNavButton>
         </Grid>
+        <Grid item xs={14} sm={2.25}>
+          <MainNavButton
+            variant="contained"
+            startIcon={<ContentPasteIcon />}
+            iconbackgroundcolour="#9F4DB7"
+            onClick={() => navigate("/job-postings/")}
+            sx={{
+              whiteSpace: "nowrap",
+            }}
+          >
+            Job Postings
+          </MainNavButton>
+        </Grid>
+        <Grid item xs={14} sm={2.25}>
+          <MainNavButton
+            variant="contained"
+            startIcon={<InboxIcon />}
+            iconbackgroundcolour="#F4B73F"
+            onClick={() => navigate("/job-apps/")}
```

Comment:
The trailing slash might cause issues - I looked at the code and all the endpoints don't have a trailing slash. I don't know if it would cause an error but removing it might be for the best. I could be wrong though!